io.typora.flatpak.desktop

```diff
@@ -0,0 +1,11 @@
+[Desktop Entry]
+Name=Typora
+Comment=A minimal Markdown reading & writing app.
```

Comment:
Desktop file comments are usually not ending with dots. Same thing applies to AppStream metadata summary.

Reply:
Hi, I pushed an update for this, thanks.